tests/lib/helpers.js

```diff
@@ -30,15 +31,34 @@ define([
     return template.replace('{id}', Math.random()) + '@restmail.net';
   }
 
+  function createPhoneNumber () {
+    // start with an area code and known good first number,
+    // append a 6 character suffix.
+    const prefix = '9164';
+    // For a total of 10 characters, we need
+    let suffix = Math.floor(Math.random() * Math.pow(10, 6));
+    suffix = padright(suffix, 6, '0');
```

Comment:
I'm not sure whether padleft or padright is better here, or whether it makes any difference at all.